#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_festival-tts-plugin.dpatch by Deng Xiyue <manphiz-guest@users.alioth.debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct path for headers.
 
@DPATCH@
diff -urNad stardict-3.0.1~/configure.in stardict-3.0.1/configure.in
--- stardict-3.0.1~/configure.in	 2007-11-01 15:37:15.000000000 +0800
+++ stardict-3.0.1/configure.in		 2008-01-07 14:03:10.000000000 +0800
@@ -130,7 +130,7 @@
 	else
 		AC_MSG_RESULT(yes)
 		FESTIVAL_LIBS="-lFestival -lestools -lestbase -leststring"
-		FESTIVAL_CFLAGS="-I/usr/include/speech-tools/EST -ffriend-injection -Wno-deprecated"
+		FESTIVAL_CFLAGS="-I/usr/include/speech_tools -ffriend-injection -Wno-deprecated"
 		FESTIVAL_PLUGIN_DIR="stardict-festival-tts-plugin"
 	fi
 else
